app.py

```diff
@@ -68,6 +68,14 @@
     _LOGGER.info("installation=%s", RUNTIME_CONTEXT.app_installation)
 
 
+# We simply extend the GitHub Event set for our use case ;)
+@process_event("thoth_thamos_advise", action="finished")
```

Comment:
It looks like you need `repository_dispatch` here: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#external-events-repository_dispatch / https://developer.github.com/v3/activity/events/types/#repositorydispatchevent / https://developer.github.com/v3/repos/#create-a-repository-dispatch-event

Reply:
ja, that might be a second possible route. We just wanted a 3rd component to be able to signal something to the webhook receiver.

Reply:
This way you could add more Apps that subscribe to the same pub-sub bus w/o any special cases...